Добавлена документация по flow работы с проектом

AntLer-24rus opened the pull request:
Сделана проверка сообщений коммитов

AntLer-24rus committed:
docs: добавлена документация по flow работы с проектом

Сделана проверка сообщений коммитов

sapomaro commented on docs/workFlow.md:
```diff
@@ -0,0 +1,95 @@
+## 1. GitHub
+
+- Создаем новую ветку, делаем туда коммиты
+- Когда все готово создаем PR в ветку dev
```
У нас эта ветка имеет название develop, нужно поменять здесь и ниже

DmitriiNazimov replied:
1. Исправил наименование ветки (dev -> develop). 
2. Добавил зависимость commitlint. Без нее сделать коммит было невозможно - yarn выдавал ошибку.

sapomaro reviewed (commented):
Нужно актуализировать название ветки, куда будут вливаться PR (dev -> develop)

DmitriiNazimov committed:
docs(workFlow.md): fix name of developing branch

dev -> develop

DmitriiNazimov commented:
1. Исправил наименование ветки (dev -> develop).
2. Добавил зависимость commitlint. Без нее сделать коммит было невозможно - yarn выдавал ошибку.

sdelfi committed:
Add react router component (#1)

* feat(router): add react router component

add basic project structure
add router error handler page
add router test

BREAKING CHANGE: change app folder structure

* refactor: replace default imports/exports with naming ones

* chore: update stylelint config

* chore(yarn): fix package version

DmitriiNazimov committed:
chore(package.json): add commitlint to devDependencies

Without commitlint it was impossible to make a commit.

DmitriiNazimov committed:
Merge branch 'develop' into task/workflow-docs

DmitriiNazimov commented:
Pull request отклоняю т.к. было указано, что он мерджится в ветку main. Сделал аналогичный pull request в ветку develop и одобрил его.
